src/Model/Traits/HasActionsTrait.php

```diff
@@ -2,11 +2,11 @@
 
 declare(strict_types=1);
 
-namespace atk4\data\Model\Concerns;
+namespace atk4\data\Model\Traits;
```

Comment:
NS should always be in singular form. Not sure if Traits trait is desired. I do not like it, also never seems it in another projects. I think Model is enought.

Reply:
Then you will have classes and traits mixed in one folder which I do not like.
I prefer separating traits in a sub-folder. `Traits` and `Concerns` I have seen but not a `Trait` subfolder

Reply:
Yes, I think no additional folder/NS is fine. We should be oriented on the use - nevel plural, never ever.

And `Concerns`? They says nothing to me, I do not like it.

Reply:
Concerns is Laravel-style

Reply:
https://stackoverflow.com/questions/40288657/how-to-make-a-trait-in-laravel - I done some research on your answer - it seems they switched from Concerns to Traits. But Without Trait suffix then.

As this clearly violates current PHP community recommendations, I am strictly against it.

Reply:
I would agree that Model NS is already suitable for the traits (of Model) so I moved them there